src/h5web/toolbar/controls/DomainSlider/DomainSlider.tsx

```diff
@@ -89,6 +87,8 @@ function DomainSlider(props: Props): ReactElement {
         iconOnly
         small
         label="Edit domain"
+        aria-expanded={hovered || isEditing}
+        aria-controls={TOOLTIP_ID}
```

Comment:
After some more research, I decided to implement the tooltip as a `dialog` rather than a `tooltip`. With the previous implementation, I could not find the parent `div` of the slider in an accessible way, so I couldn't test `aria-expanded`.